### PR TITLE
NEPT-228: Editorial team member user role inconsistency

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: tests/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -505,4 +505,17 @@ public function iShouldNotSeeHighlightedElements() {
...
   }
 
+  /**
+   * Assert that the given form element is disabled.
+   *
+   * @Then the :label checkbox should be disabled
+   * @Then the :label form element should be disabled
+   */
+  public function assertDisabledElement($label) {
+    $checkbox = $this->getSession()->getPage()->findField($label);
```
Comment (voidtek) [inline tests/features/bootstrap/FeatureContext.php]: Could check if $checkbox is find?

File: tests/features/editorial_workflow.feature [hunk 2]
```diff
@@ -96,3 +96,12 @@ Feature: Editorial workflow
       | editor             |
       | contributor        |
       | authenticated user |
+
+  Scenario: User role "editorial team member" cannot be assigned via the UI.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/people"
+    Then I should not have the following options for "edit-operation":
+      | editorial team member |
```
Comment (voidtek) [inline tests/features/editorial_workflow.feature]: The first line of this table should be the title:

example:
      | options                          |
      | editorial team member |

File: tests/features/editorial_workflow.feature [hunk 3]
```diff
@@ -96,3 +96,12 @@ Feature: Editorial workflow
       | editor             |
       | contributor        |
       | authenticated user |
+
+  Scenario: User role "editorial team member" cannot be assigned via the UI.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/people"
+    Then I should not have the following options for "edit-operation":
```
Comment (voidtek) [inline tests/features/editorial_workflow.feature]: This function only checks on the first level of the option ("editorial team member" is in a second level of options)
It doesn't check in "optgroup".

File: tests/features/editorial_workflow.feature (hunk 2 above)
Comment (ademarco) [inline tests/features/editorial_workflow.feature]: Done, thanks, I've added documentation so in future we know what to do.

File: tests/features/bootstrap/FeatureContext.php [hunk 4]
```diff
@@ -505,4 +505,17 @@ public function iShouldNotSeeHighlightedElements() {
...
+    if (!$checkbox->hasAttribute('disabled')) {
```
Comment (ademarco) [inline tests/features/bootstrap/FeatureContext.php]: Fixed.

File: tests/features/bootstrap/FeatureContext.php (hunk 1 above)
Comment (ademarco) [inline tests/features/bootstrap/FeatureContext.php]: Fixed.

File: tests/features/editorial_workflow.feature [hunk 5]
```diff
@@ -96,3 +96,12 @@ Feature: Editorial workflow
...
+      | editorial team member |
+    And I am on "user"
```
Comment (ademarco) [inline tests/features/editorial_workflow.feature]: Done.

File: tests/features/editorial_workflow.feature (hunk 3 above)
Comment (voidtek) [inline tests/features/editorial_workflow.feature]: perfect, thanks.

**Comment by dxvargas** [comment]
👍

**Comment by voidtek** [comment]
:+1:

**Pull request merged**